src/OpenFeature.DependencyInjection/OpenFeatureBuilderExtensions.cs

```diff
@@ -0,0 +1,270 @@
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Options;
+using OpenFeature.DependencyInjection;
+using OpenFeature.Model;
+
+namespace OpenFeature;
+
+/// <summary>
+/// Contains extension methods for the <see cref="OpenFeatureBuilder"/> class.
+/// </summary>
+#if NET8_0_OR_GREATER
+[System.Diagnostics.CodeAnalysis.Experimental(DependencyInjection.Diagnostics.FeatureCodes.NewDi)]
+#endif
+public static partial class OpenFeatureBuilderExtensions
+{
+    /// <summary>
+    /// This method is used to add a new context to the service collection.
+    /// </summary>
+    /// <param name="builder">The <see cref="OpenFeatureBuilder"/> instance.</param>
```

Comment:
nit: It would be nice to keep the style of parameter documentation consistent. I think that the prevalent style is currently fragments instead of sentences.